src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {render} from 'react-dom'
+import './index.css';
+import App from './App';
+import store from './store';
+import actions from './actions'
+import {Provider} from 'react-redux'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import books from './data/books'
+store.dispatch(actions.listBooksAction(books))
```

Comment:
Listing books is out of index.js scope, it should not know about books or anything related.
BookList component is the one that needs access to the list of books.

1. Connect _BookList_ like _BookListItem_ component to redux
2. Use _mapDispatchToProps_ method to access the one related to listing books
3. Pick proper lifecycle event to call it, I'd suggest _componentDidMount_

Reply:
@lebaz20 So I will convert BookList component to be class instead of function, So I can use lifecycle events... Right?